lib/bitcrowd_ecto/assertions.ex

```diff
@@ -325,16 +325,18 @@ defmodule BitcrowdEcto.Assertions do
   @doc since: "0.1.0"
   @spec assert_count_difference(Ecto.Repo.t(), module, integer, (() -> any)) ::
           Changeset.t() | no_return
-  def assert_count_difference(repo, schema, by, how) do
-    assert_difference(fn -> repo.count(schema) end, by, how,
+  @spec assert_count_difference(Ecto.Repo.t(), module, integer, (() -> any), keyword) ::
+          Changeset.t() | no_return
+  def assert_count_difference(repo, schema, by, how, opts \\ []) do
```

Comment:
I think I'd prefer it if the `opts` were the fourth argument, i.e. before the function. Functions we pass are often long, and the trailing `end), prefix: "foo")` looks a bit funny?

Reply:
I agree the trailing prefix looks funny, but also might be rarely used? also, if we put opts in the middle we lose our last argument sugar. 
the more I think about this though, the more I'm okay with it. I'll give it a try.. see how it looks

Reply:
ok, also didn't know about these :)